app/src/main/java/com/keylesspalace/tusky/components/timeline/viewmodel/CachedTimelineViewModel.kt

```diff
@@ -81,6 +84,16 @@ class CachedTimelineViewModel @Inject constructor(
         }
         .cachedIn(viewModelScope)
 
+    init {
+        viewModelScope.launch {
+            delay(5.toDuration(DurationUnit.SECONDS)) // delay so the db is not locked during initial
```

Comment:
So the cleanup will run on every account switch in Tusky, maybe that is too often? But it won't have to clean that much when it runs often

Reply:
I don't know if it's good that viewModel does such cleanup tasks but otherwise I think it's totally fine to do it every time